builtin/providers/aws/resource_aws_vpc_dhcp_options.go

```diff
@@ -121,7 +121,7 @@ func resourceAwsVpcDhcpOptionsCreate(d *schema.ResourceData, meta interface{}) e
 	log.Printf("[DEBUG] Waiting for DHCP Options (%s) to become available", d.Id())
 	stateConf := &resource.StateChangeConf{
 		Pending: []string{"pending"},
-		Target:  "",
+		Target:  []string{},
```

Comment:
There are a handful of these `Target: []string{}` that I think have broken stuff (see https://travis-ci.org/hashicorp/terraform/jobs/104548430). I'm fixing, but was wondering if you hit this in development and possibly there's a bug elsewhere